wger/gym/templates/gym/member_list.html

```diff
@@ -55,7 +56,17 @@
     <td>
         {{current_user.obj.get_full_name}}
     </td>
-
+    {% if perms.gym.manage_gym or perms.gym.manage_gyms %}
+        <td>
+            {% if current_user.obj.id != user.id %}
```

Comment:
try combine line 61 and 62

Reply:
are there tests for this functionality ???

Reply:
The reason for the two if statements is that the first allows for if the user is a gym manager giving him/ her access to the deactivate tab while the second gives the option of rejecting the current user from deactivating his/ her own account. Also allows for UI inconsistencies in the table
The tests already their for the functions I called and did not see the need to alter them.